src/ucs/sys/sys.c

```diff
@@ -442,6 +443,28 @@ size_t ucs_get_phys_mem_size()
     return phys_pages * ucs_get_page_size();
 }
 
+#define UCS_SHMMAX "/proc/sys/kernel/shmmax"
```

Comment:
pls rename to UCS_PROC_SYS_SHMMAX_FILE and put in top of the file / or remove the constant